pkg/config/config.go

```diff
@@ -103,6 +103,15 @@ func LoadOrCreate() (*CLIConfig, error) {
 	return c, nil
 }
 
+func (c *CLIConfig) ProjectExists(name string) bool {
+	for _, p := range c.Projects {
+		if p.Name == name {
```

Comment:
can we do a case insensitive comparison? just in case.

Reply:
yes, good call!